src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,42 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Card {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+type List = {
+  movies: Card[],
+};
+
+export const MoviesList: React.FC<List> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(({
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+    }) => {
+      return (
+        <div
+          className="card"
+          key={imdbId}
+          data-cy="card"
+        >
```

Comment:
- main `div` `MovieCard` component should have a `data-cy="card"` attribute
not in MovieList